### DISCUSSION TITLE
Docs: better nesting

### DISCUSSION THREAD

**Discussion by pivaszbs** [reatom/reatom#789] [Ideas]
For example it's too hard to read about reatom/core beacuse it's wall of text
If it will be different pages like

- atom
   - simple
   - advanced
- action
   - simple
   - advanced
- ctx
Everything step by stem from easiest one to advanced

- mainIdeas behind
Some core principles of how reatom working

It will be easier, beause right now it's hard to understand))

**Comment by BANOnotIT** [comment]
Would simple TOC solve this navigation problem?

**Comment by BANOnotIT** [comment]
Pls, check out new site. Is it ok now?
